Formula/nmh.rb

```diff
@@ -0,0 +1,35 @@
+class Nmh < Formula
+  desc "Nmh, the new version of the MH mail handler"
+  homepage "http://www.nongnu.org/nmh/"
+  url "https://download.savannah.gnu.org/releases/nmh/nmh-1.6.tar.gz"
+  sha256 "29338ae2bc8722fe8a5904b7b601a63943b72b07b6fcda53f3a354edb6a64bc3"
+
+  depends_on "openssl"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}", "--libdir=#{libexec}",
+                          "--with-cyrus-sasl",
+                          "--with-tls"
+    system "make", "install"
+  end
+
+  test do
+    Dir.mkdir(testpath/"Mail")
+    Dir.mkdir(testpath/"Mail/inbox")
+    (testpath/".mh_profile").write "Path: Mail"
+    (testpath/"Mail/inbox/1").write <<-EOM.undent
+      From: Mister Test <test@example.com>
+      To: Mister Nobody <nobody@example.com>
+      Date: Tue, 5 May 2015 12:00:00 -0000
+      Subject: Hello!
+
+      How are you?
+    EOM
+    ENV["TZ"] = "GMT"
+    output = `#{bin}/scan -width 80`
```

Comment:
We need to improve our automated tests for this (e.g. `brew audit`) but thanks for the suggestion.